core/templates/public_layout.html

```diff
@@ -44,25 +44,27 @@
   <body>
     {% include "navbar.html" with show_main_header=True %}
 
-    <div class="row">
-      <div class="content-wrap col-md-offset-1 col-md-10">
-        {% if redirect_field_value %}
-          <div>
-            <div class="col-md-12">
-              <div class="alert alert-info alert-dismissable">
-                <i class="icon-exclamation-sign"></i>
-                {% trans "You need to be logged in to continue to the next step." %}
-                <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&#215;</button>
-              </div>
-            </div>
-          </div>
-        {% endif %}
-
-        {% bootstrap_messages messages %}
-
-        {% block content %}{% endblock content %}
-      </div>
-    </div>
+    <div id="pad-wrapper">
```

Comment:
You need to fix this indentations. I recommend you to Spaces instead of Tabs on your text editor.

Reply:
And that `pad-wrapper` class, I think that you need to commit a `.css` file, right? Or it is a class from Bootstrap?

Reply:
Another thing, can you add a printscreen from the "Signup" and "Events" pages? For to look how its changing the layout.

Reply:
I fixed the indentation.
The css class already exists in the project, there was no need for inclusion.
See the result:
![print1](https://user-images.githubusercontent.com/11786543/31785811-1f2adac6-b4e5-11e7-8a79-68a1208e0de8.png)
![print2](https://user-images.githubusercontent.com/11786543/31785812-1f6e2b3c-b4e5-11e7-9aa9-1a880aaa255c.png)